package.json

```diff
@@ -6,14 +6,20 @@
   "license": "Apache 2",
   "repository": "https://github.com/coinalpha/gateway-api",
   "scripts": {
-    "start": "pm2 start babel-node --no-daemon -- src/index.js",
+    "prebuild": "rimraf dist && mkdir dist",
+    "build": "tsc --project ./",
+    "format": "prettier . --write",
+    "lint": "eslint src test --format table --fix",
+    "debug": "DEBUG=*router nodemon --ext 'ts,js,json' --exec 'ts-node' src/index.ts",
+    "start": "pm2 start babel-node --no-daemon -- dist/index.js",
```

Comment:
Why you using pm2?
pm2 is great library, but I'm not sure that we really need to use it.